src/test/resources/specs/name-matching/Name_7Strings_HMRC.feature

```diff
@@ -0,0 +1,133 @@
+Feature: Name matching with 7 name strings
+
+  Scenario: applicant with 7 names is matched on the last combination
```

Comment:
Can we also have one of these for 6 names, just so it is covered off.

Reply:
Done - added to a new file `Name_6Strings_HMRC.feature`